samples/bluetooth/hci_uart/src/main.c

```diff
@@ -317,7 +317,7 @@ static int hci_uart_init(struct device *unused)
 	SYS_LOG_DBG("");
 
 	hci_uart_dev =
-		device_get_binding(CONFIG_BLUETOOTH_UART_TO_HOST_DEV_NAME);
+	device_get_binding(CONFIG_BLUETOOTH_CONTROLLER_TO_HOST_UART_DEV_NAME);
```

Comment:
Just keep the original indent. Going over the 80 char width is a smaller offence than making it look like we're making a function call who's return value we're not storing anywhere (because the call is at the default indent of the function)